README.md

```diff
@@ -77,6 +84,20 @@ cargo clippy
 cargo test
 ```
 
+### VS Code support
+
+Recommended extensions are defined in `.vscode/extensions.json`. VS Code normally prompts you and offers you to install any missing recommended extensions. In case VS Code does not prompt you, you can install the recommended extensions manually. Use the extension IDs from `extensions.json` to search for and install extensions.
+
+Remember to run `npm install` in the project root to install Prettier. Otherwise, the Prettier extension will not work. When you add
+
+```json
+{
+  "editor.formatOnSave": true
+}
+```
+
+to your user settings, VS Code will automatically format frontend and JSON files when you hit save.
```

Comment:
also here user settings should, in my opinion, just be a clear file path : `.vscode/settings.json` to be explicit.